examples/common.h

```diff
@@ -31,6 +31,7 @@ struct gpt_params {
     int32_t main_gpu                        = 0;   // the GPU that is used for scratch and small tensors
     float   tensor_split[LLAMA_MAX_DEVICES] = {0}; // how split tensors should be distributed across GPUs
     bool    low_vram                        = 0;   // if true, reduce VRAM usage at the cost of performance
+    int32_t n_probs                         = 0;   // if greater than 0, output the probabilities of top n_probs tokens.
```

Comment:
It should probably not even be in common.h. The server has its own parameter struct in server.cpp.

Reply:
I put it in common.h because I thought other examples might use n_probs for custom stop sequences too. Like a chatbot.